### PR TITLE
feat: added all the back arrows on screens that needed it

### PR DESCRIPTION
**Pull request by GaelCondeLosada**
# What I did
Added back arrows on all screen that were missing it.

# How I did it
- Added the component `BackArrow` in the screens.
- Adapted the tests.

# How to verify it
- Open the app and check that the back arrow is on all screens that have one on the figma and that the arrow works correctly.
- You can also run the tests.

# Demo video

# Pre-merge checklist
The changes I have introduced:
- [x] work correctly
- [x] do not break other functionalities
- [x] work correctly on Android
- [x] are fully tested

### CODE REVIEW

File: screens/Profile/ExternalProfileScreen/ExternalProfileScreen.tsx [hunk 1]
```diff
@@ -77,6 +78,7 @@ const ExternalProfileScreen = () => {
 
   return (
     <View style={styles.container}>
+      <BackArrow/>
```
Comment (alexandre-mrt) [inline screens/Profile/ExternalProfileScreen/ExternalProfileScreen.tsx:81]: When testing with the app it doesn't render correctly. You should try putting it outside of the container. Moreover the position is absolute so take care about overlapping on layouts

**Review by alexandre-mrt — CHANGES_REQUESTED** [review]
LGTM. Just for the external profile try to make the layout great again

File: screens/Profile/ExternalProfileScreen/ExternalProfileScreen.tsx (hunk 1 above)
Comment (GaelCondeLosada) [reply screens/Profile/ExternalProfileScreen/ExternalProfileScreen.tsx:81]: There is another PR opened to fix this problem (#243 )

**Review by alexandre-mrt — APPROVED** [review]
LGTM

**Pull request merged**